_about/notes/0.6.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Istio 0.6
+
+order: 95
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Custom Envoy Config**. Pilot now supports ferrying cuatom Envoy config to the
+proxy. TBD: link?
+
+- **External Services**. TBD: what can we say here?
```

Comment:
Rather than calling out external services + gateways, lets just call out the entire new API surface. @rshriram @louiscryan @frankbu What kind of a blurb do we want to describe the changes from v1alpha1 to v1alpha3?

(BTW we should note somewhere in these release notes that we're skipping v1alpha2 due to CRD issues.)

Reply:
I agree with @ZackButcher that we should call out the entire model, but since it's not really working yet, shouldn't we wait until 0.7? I was imagining that we will update the samples, tasks, and overview doc (https://istio.io/docs/concepts/traffic-management/rules-configuration.html) to the new model, and add a blog that talks about moving from v1alpha1 to v1alpha3, but first step is to get the samples converted.

Reply:
Ah yes, that's what we did in the previous cycle too. OK, I removed the External Service and Gateway items and hopefully they'll come back with a roar next month.

Reply:
AFAIK we are planning to ship the CRDs by default with 0.6; you'll be able to use both old and new side-by-side for at least this next release.

Or are we planning to punt this till 0.7 too?